From 27264eae807d894ba0f889e90756281194855540 Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Sun, 11 Oct 2020 11:02:35 -0700
Subject: [PATCH] * ziguard.awk: Add limitations commentary.

---
 ziguard.awk | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/ziguard.awk b/ziguard.awk
index e8ef49e..e27e799 100644
--- a/ziguard.awk
+++ b/ziguard.awk
@@ -3,6 +3,13 @@
 # Contributed by Paul Eggert.  This file is in the public domain.
 
 # This is not a general-purpose converter; it is designed for current tzdata.
+# It just converts from current source to main, vanguard, and rearguard forms.
+# Although it might be nice for it to be idempotent, or to be useful
+# for converting back and forth between vanguard and rearguard formats,
+# it does not do these nonessential tasks now.
+#
+# Although main and vanguard forms are currently equivalent,
+# this need not always be the case.
 #
 # When converting to vanguard form, the output can use negative SAVE
 # values.
@@ -28,7 +35,7 @@ DATAFORM != "main" {
   in_comment = /^#/
   uncomment = comment_out = 0
 
-  # If the line should differ due to Czechoslovakia using negative SAVE values,
+  # If this line should differ due to Czechoslovakia using negative SAVE values,
   # uncomment the desired version and comment out the undesired one.
   if (zone == "Europe/Prague" && /1947 Feb 23/) {
     if (($(in_comment + 2) != "-") == vanguard) {
-- 
1.8.3.1

