From 9c175ec6c48c9292a2ee1ed285e0f492a2d5a13a Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Wed, 21 Oct 2020 11:24:13 -0700
Subject: [PATCH] Release 2020d

* NEWS: Increase version to 2020d.
---
 NEWS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/NEWS b/NEWS
index 66a579d..328face 100644
--- a/NEWS
+++ b/NEWS
@@ -1,6 +1,6 @@
 News for the tz database
 
-Unreleased, experimental changes
+Release 2020d - 2020-10-21 11:24:13 -0700
 
   Briefly:
     Palestine ends DST earlier than predicted, on 2020-10-24.
-- 
1.8.3.1

