From 243321439b4f47dbc6981c990fa6eb3b94a01e98 Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Tue, 18 Aug 2020 20:47:17 -0700
Subject: [PATCH 27/47] * README: Add a pointer to tz-how-to.html.

---
 README | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/README b/README
index dd6fcf7..145aacd 100644
--- a/README
+++ b/README
@@ -20,6 +20,8 @@ substituting your desired installation directory for "$HOME/tzdir":
 	make TOPDIR=$HOME/tzdir install
 	$HOME/tzdir/usr/bin/zdump -v America/Los_Angeles
 
+See the file tz-how-to.html for examples of how to read the data files.
+
 This database of historical local time information has several goals:
 
  * Provide a compendium of data about the history of civil time that
-- 
1.8.3.1

