From 3e865829ebbb6639d3392f4a10a7c192572be833 Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Sun, 23 Aug 2020 15:59:39 -0700
Subject: [PATCH 29/47] * asia: Cite Barak 2020 on Israeli DST controversy.

---
 asia | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/asia b/asia
index 106efad..fff9a10 100644
--- a/asia
+++ b/asia
@@ -1699,6 +1699,10 @@ Zone	Asia/Baghdad	2:57:40	-	LMT	1890
 
 # Israel
 
+# For more info about the motivation for DST in Israel, see:
+# Barak Y. Israel's Daylight Saving Time controversy. Israel Affairs.
+# 2020-08-11. https://doi.org/10.1080/13537121.2020.1806564
+
 # From Ephraim Silverberg (2001-01-11):
 #
 # I coined "IST/IDT" circa 1988.  Until then there were three
-- 
1.8.3.1

