From c9e3778253448f5c4b2f3f64e00bcf27fc38d778 Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Fri, 12 Jun 2020 15:20:56 -0700
Subject: [PATCH 14/47] * europe: Fix minor comment typos in previous change.

---
 europe | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/europe b/europe
index e8ec780..4dc159f 100644
--- a/europe
+++ b/europe
@@ -2093,8 +2093,8 @@ Zone	Europe/Chisinau	1:55:20 -	LMT	1880
 # In the "Journal de Monaco" of 1911-03-28, online at
 # https://journaldemonaco.gouv.mc/var/jdm/storage/original/application/de74ffb7db53d4f599059fe8f0ed482a.pdf
 # we read an ordinance of 1911-03-16: ...
-#  [Legal time in the Pricipality will be set, from the date of promulgation
-#   of the present ordinance, to legal time in France.  Consequently, legal
+#  [Legal time in the Principality will be set, from the date of promulgation
+#   of the present ordinance, to legal time in France....  Consequently, legal
 #   time will be retarded by 9 minutes and 21 seconds.]
 #
 # Zone	NAME		STDOFF	RULES	FORMAT	[UNTIL]
-- 
1.8.3.1

