From 3dcdea78d28b47f54dd15ca48207119a441b176b Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Mon, 7 Dec 2020 11:37:59 -0800
Subject: [PATCH] Fix NEWS typo

---
 NEWS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/NEWS b/NEWS
index bd9a2d2..041d224 100644
--- a/NEWS
+++ b/NEWS
@@ -4,7 +4,7 @@ Unreleased, experimental changes
 
   Changes to future timestamps
 
-    Volgograd changes time zone from +04 to +03 on 2020-12-20 at 02:00.
+    Volgograd changes time zone from +04 to +03 on 2020-12-27 at 02:00.
     (Thanks to Alexander Krivenyshev and Stepan Golosunov.)
 
   Changes to past timestamps
-- 
1.8.3.1

