From a499dd6e0836ba149490ca0491aa5363780cdd18 Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Wed, 25 Nov 2020 10:17:16 -0800
Subject: [PATCH] * northamerica: Add URL for Yukon OIC 1980/02

---
 northamerica | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/northamerica b/northamerica
index d5abf21..b9a1348 100644
--- a/northamerica
+++ b/northamerica
@@ -2234,7 +2234,7 @@ Zone America/Creston	-7:46:04 -	LMT	1884
 #     to say eight hours behind Greenwich Time.
 #
 # * O.I.C. 1980/02 INTERPRETATION ACT
-#   [no online source found]
+#   https://mm.icann.org/pipermail/tz/attachments/20201125/d5adc93b/CAYTOIC1980-02DST1980-01-04-0001.pdf
 #
 # * Yukon Daylight Saving Time, YOIC 1987/56
 #   https://www.canlii.org/en/yk/laws/regu/yoic-1987-56/latest/yoic-1987-56.html
-- 
1.8.3.1

