From efc5c061920f1d5820d66b7ec88ebff144c55b3f Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Tue, 22 Dec 2020 15:14:34 -0800
Subject: [PATCH] Release 2020e

* NEWS: Increase version to 2020e.
---
 NEWS | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/NEWS b/NEWS
index 991b737..6014b45 100644
--- a/NEWS
+++ b/NEWS
@@ -1,6 +1,9 @@
 News for the tz database
 
-Unreleased, experimental changes
+Release 2020e - 2020-12-22 15:14:34 -0800
+
+  Briefly:
+    Volgograd switches to Moscow time on 2020-12-27 at 02:00.
 
   Changes to future timestamps
 
-- 
1.8.3.1

