diff --git a/asia b/asia
index 106efad..c631935 100644
--- a/asia
+++ b/asia
@@ -675,6 +675,8 @@ Zone	Asia/Shanghai	8:05:43	-	LMT	1901
 Zone	Asia/Urumqi	5:50:20	-	LMT	1928
 			6:00	-	+06
 
+# Nowadays, China Standard Time is known as "Beijing Time"
+Zone    Asia/Beijing    8:00    PRC     C%sT
 
 # Hong Kong
 
diff --git a/backward b/backward
index e13ae52..348f99e 100644
--- a/backward
+++ b/backward
@@ -101,7 +101,7 @@ Link	America/Mexico_City	Mexico/General
 Link	Pacific/Auckland	NZ
 Link	Pacific/Chatham		NZ-CHAT
 Link	America/Denver		Navajo
-Link	Asia/Shanghai		PRC
+Link	Asia/Beijing		PRC
 Link	Pacific/Kanton		Pacific/Enderbury
 Link	Pacific/Honolulu	Pacific/Johnston
 Link	Pacific/Pago_Pago	Pacific/Midway
diff --git a/zone.tab b/zone.tab
index 8d056e3..cfd90aa 100644
--- a/zone.tab
+++ b/zone.tab
@@ -145,7 +145,8 @@ CL	-3327-07040	America/Santiago	Chile (most areas)
 CL	-5309-07055	America/Punta_Arenas	Region of Magallanes
 CL	-2709-10926	Pacific/Easter	Easter Island
 CM	+0403+00942	Africa/Douala
-CN	+3114+12128	Asia/Shanghai	Beijing Time
+CN	+3955+11626	Asia/Beijing	China Standard Time
+CN	+3114+12128	Asia/Shanghai	Shanghai Time
 CN	+4348+08735	Asia/Urumqi	Xinjiang Time
 CO	+0436-07405	America/Bogota
 CR	+0956-08405	America/Costa_Rica
diff --git a/zone1970.tab b/zone1970.tab
index 53ee77e..3d21c92 100644
--- a/zone1970.tab
+++ b/zone1970.tab
@@ -137,7 +137,8 @@ CK	-2114-15946	Pacific/Rarotonga
 CL	-3327-07040	America/Santiago	Chile (most areas)
 CL	-5309-07055	America/Punta_Arenas	Region of Magallanes
 CL	-2709-10926	Pacific/Easter	Easter Island
-CN	+3114+12128	Asia/Shanghai	Beijing Time
+CN	+3955+11626	Asia/Beijing	China Standard Time
+CN	+3114+12128	Asia/Shanghai	Shanghai Time
 CN	+4348+08735	Asia/Urumqi	Xinjiang Time
 CO	+0436-07405	America/Bogota
 CR	+0956-08405	America/Costa_Rica
